DTS-STN/dts-rmp
# 132 update nav buttons

# Description

Modified the navigation of each buttons/nuxt-links to use the proper pages inside the 'pages' folder. 

## Test Instructions

Navigate through the website.
Use the orange nav buttons and the white and blue contact/engagement toggles.
Notice the URL as it should be redirected to their proper pages.

make sure to refresh the page and investigate if the buttons are respectively selected.

## Help Requested

Important: 
If you click to go to the home page, the orange nav button will still be selected. This is on purpose since I did not know if we are going to set the home to search by default.

## Checklist
